slave/process-eduroam-block/bin/process-eduroam-block.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+PROTOCOL_VERSION='3.0.0'
+
+# Save file with blocked users
+function process {
+
+	FROM_PERUN="${WORK_DIR}/eduroam_block"
+
+	DST_DIR="/etc/eduroam"
```

Comment:
Correct folder should be ``/etc/raddb/`` and file name ``blacklist``, but since we probably won't have root access to the machine, we will drop file somewhere else and they will move it to correct location and restart radius server. I 'll let you know once I will get details.